src/main/openapi/tenant-svc-openapi-internal.yaml

```diff
@@ -5,45 +5,131 @@ info:
 servers:
   - url: https://localhost:8080
 tags:
-  - name: Customer
-    description: Customer specific data.
+  - name: TenantInternal
+    description: Execute tenant CRUD operations
 paths:
-  /customer:
+  /internal/tenant:
+      get:
+        tags:
+          - TenantInternal
+        summary: Get all Tenant maps by page number and page size
+        operationId: getAllTenantMapsPageable
```

Comment:
Is this our agreed postfix? For me, this would only make sense if there would also be an endpoint that would deliver the data in a non-paged form